### PR TITLE
Finalize seeding data and importing data to MySQL database

### PR DESCRIPTION
**Pull request by katie-truong**
Seeding data directly to MySQL without converting it to CSV

### CODE REVIEW

**Comment by katie-truong** [comment]
Hi @courtchao can you review this PR? :)

File: db/item-generator.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+});
+
+// Use the newly-created database
+connection.query('use expressel;', (error) => {
+  if (error) throw error;
+});
+
+// Create table 'items'
+connection.query(`create table if not exists items (
+  id integer not null auto_increment,
```
Comment (courtchao) [inline db/item-generator.js]: Small thing, but would be nice to have everything uniform (either all int or all integer)

File: db/item-generator.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+const randomItemPicker = function(array) {
+  return array[generateRandomNumber(0, array.length)];
+};
+
+// Use to generate a random list of items
+const generateItems = function(num) {
+  for (let i = 0; i < num; i++) {
+    let generatedItem = {
+      price: generateRandomNumber(1, 3000, 2),
+      storeID: generateRandomNumber(1, 5, 0),
```
Comment (courtchao) [inline db/item-generator.js]: Will storeID and storeName not match up? Not blocking issue for now, but curious why?

File: db/item-generator.js (hunk 1 above)
Comment (katie-truong) [reply db/item-generator.js]: thanks for pointing it out!

File: db/item-generator.js (hunk 2 above)
Comment (katie-truong) [reply db/item-generator.js]: oh thanks for pointing it out! will fix it!

**Pull request closed without merging**